### PR TITLE
Document where to find other images used for testing

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -112,6 +112,18 @@ tests/test_default.yml`
...
+To use the CentOS 6, CentOS 7 images, please checkout the link below:  
+
+https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-2003.qcow2c
+
+- `7/images/` stands for CentOS 7 images, we can also change it to `6/images/` to use
+CentOS 6 images.
+- For above qe testing cases, we prefer the image architecture to be `x86_64-GenericCloud`.
+- `2003` in `CentOS-7-x86_64-GenericCloud-2003.qcow2c` stands for image released in
+March 2020. 
+- We can use the image with extension `.qcow2` and `.qcow2c`.
```
Comment (tyll) [inline contributing.md:125]: Great explanation, please also add the actual URLs to the directories for CentOS 6 and 8. Also, please mention Fedora as well.

Comment (liangwen12year) [reply contributing.md]: @tyll , I do not know where to find the Fedora images

Comment (richm) [reply contributing.md]: Not sure how we should document that.  The main page for getting Fedora is https://getfedora.org/ - but it does not have links specifically for the type of cloud images we use for integration/qemu testing.  These are listed under Fedora Alternative Downloads, then under Cloud Base Images, then under https://alt.fedoraproject.org/cloud/, then under Cloud Base image for Openstack
We could also provide a link directly to the download site e.g. http://mirrors.kernel.org/fedora/releases/$FEDORA_VERSION/Cloud/x86_64/images/Fedora-Cloud-Base-$FEDORA_VERSION-$ver.x86_64.qcow2

**Review by ffmancera — CHANGES_REQUESTED** [review]
Please, add the Fedora images as @richm suggested on: https://github.com/linux-system-roles/network/pull/293#discussion_r496051319

File: contributing.md (hunk 1 above)
Comment (liangwen12year) [reply contributing.md]: updated

**Review by ffmancera — CHANGES_REQUESTED** [review]
Fedora 31 is also needed. Could you please add it? In addition, DCO bot is failing. Could you please check if your commit is signed-off correctly? Thanks, Fernando!

**Comment by liangwen12year** [comment]
> Fedora 31 is also needed. Could you please add it? In addition, DCO bot is failing. Could you please check if your commit is signed-off correctly? Thanks, Fernando!

updated. I checked, my commit is signed-off correctly.

File: contributing.md (hunk 1 above)
Comment (richm) [reply contributing.md]: Even better - for Fedora, we should recommend to use the latest qcow2 images from https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-31/compose/Cloud/x86_64/images/ and https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-32/compose/Cloud/x86_64/images/.

Comment (liangwen12year) [reply contributing.md]: updated

**Review by ffmancera — APPROVED** [review]
Looks good to me, please rebase it :-)

File: contributing.md [hunk 2]
```diff
@@ -112,6 +112,25 @@ tests/test_default.yml`
...
+CentOS 7 images, change it to `/8/images/` to use CentOS 8 images.
+- For qemu testing cases, we prefer the image architecture to be `x86_64-GenericCloud`.
+- `2003` in `CentOS-7-x86_64-GenericCloud-2003.qcow2c` stands for image released in
+March 2020.
+- We can use the image with extension `.qcow2` and `.qcow2c`.
+- To save the image, right click on the link above, then select "Save link as...".
+
+For Fedora, we recommend to use the latest qcow2 images from<br/>
+http://mirrors.kernel.org/fedora/releases/31/Cloud/x86_64/images/Fedora-Cloud-Base-31-1.9.x86_64.qcow2<br/>
+http://mirrors.kernel.org/fedora/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2
```
Comment (pcahyna) [inline contributing.md:131]: The Fedora 31 one is almost a year old, so "latest" is an overstatement here. Could one use images from https://kojipkgs.fedoraproject.org/compose/cloud/ ?

Comment (liangwen12year) [reply contributing.md:131]: I compared the image download from http://mirrors.kernel.org/fedora/releases/31/Cloud/x86_64/images/Fedora-Cloud-Base-31-1.9.x86_64.qcow2, http://mirrors.kernel.org/fedora/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2, and https://kojipkgs.fedoraproject.org/compose/cloud/ while testing on some playbooks, they all failed. Can anyone nominate a testing playbooks working on Fedora ?

Comment (richm) [reply contributing.md:131]: works for me
```
curl -o /path/to/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2 https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-31/compose/Cloud/x86_64/images/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2
cd network/tests
ANSIBLE_STDOUT_CALLBACK=debug TEST_SUBJECTS=/path/to/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2 ansible-playbook -vv -i /usr/share/ansible/inventory/standard-inventory-qcow2 tests_ethtool_features_nm.yml
....
PLAY RECAP *********************************************************************
/path/to/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2 : ok=101  changed=26   unreachable=0    failed=0    skipped=39   rescued=1    ignored=0
```

Comment (liangwen12year) [reply contributing.md:131]: @pcahyna  the image from https://kojipkgs.fedoraproject.org/compose/cloud/  also works

Comment (ffmancera) [reply contributing.md:131]: @liangwen12year IMO, we should use the image from https://kojipkgs.fedoraproject.org/compose/cloud/

Comment (liangwen12year) [reply contributing.md]: @ffmancera updated the commit

**Comment by liangwen12year** [comment]
Got it.

Thanks and regards,

Wen

**Pull request merged**